index.bs

```diff
@@ -986,191 +992,191 @@ The "Version(s)" column in the following table lists the version(s) of the boxes
             <td>dinf</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>meta</td>
             <td>Used to indicate the location of the media information in a track</td>
         </tr>
         <tr>
             <td>&nbsp;</td>
             <td>dref</td>
             <td>0</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>&nbsp;</td>
             <td>&nbsp;</td>
         </tr>
         <tr>
             <td>iref</td>
             <td>&nbsp;</td>
             <td>0, 1</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>meta</td>
             <td>Used to indicate directional relationships between images or metadata</td>
         </tr>
         <tr>
             <td>&nbsp;</td>
             <td>auxl</td>
             <td>-</td>
-            <td>HEIF</td>
+            <td>[[!HEIF]]</td>
             <td>&nbsp;</td>
             <td>Used when an image is auxiliary to another image</td>
         </tr>
         <tr>
             <td>&nbsp;</td>
             <td>thmb</td>
             <td>-</td>
-            <td>HEIF</td>
+            <td>[[!HEIF]]</td>
             <td>&nbsp;</td>
             <td>Used when an image is a thumbnail of another image</td>
         </tr>
         <tr>
             <td>&nbsp;</td>
             <td>dimg</td>
             <td>-</td>
-            <td>HEIF</td>
+            <td>[[!HEIF]]</td>
             <td>&nbsp;</td>
-            <td>Used when an image is derived from another image</td>
+            <td>Used when an image is [[#derived-images|derived from another image]]</td>
```

Comment:
FYI: I noticed in the Bikeshed spec that you can provide alternative text for links by using `[=xxx|This is my alt text=]`